telegram/bot.py

```diff
@@ -154,6 +154,8 @@ def decorator(self, *args, **kwargs):
             if result is True:
                 return result
 
+            self.create_references(result)
+
```

Comment:
I think we should look for another way. This change actually changes the behaviour of a critical path in "production" code and only for tests.